code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -379,9 +509,12 @@
 	if(!message)
 		return
 	var/my_message
-	my_message = "<FONT size = 3><b>COMMAND FROM SPIDER QUEEN:</b> [message]</FONT>"
+	my_message = "<span class='spider'><b>Command from [user]:</b> [message]</span>"
 	for(var/mob/living/simple_animal/hostile/poison/giant_spider/M in GLOB.spidermobs)
 		to_chat(M, my_message)
+	for(var/M in GLOB.dead_mob_list)

```

Comment:
I'm sure there's a helper for sending messages to ghosts.

Reply:
i've been looking, but this is the only method i've seen, and it's in cult comms, abductor tongue, and clock comms sorta do this since they cycle the full mob list and check who's dead and who's not